core/domain/state_domain.py

```diff
@@ -1423,6 +1443,25 @@ def convert_state_dict_to_yaml(cls, state_dict, width):
 
         return utils.yaml_from_dict(state.to_dict(), width=width)
 
+    def get_translation_counts(self):
+        """Return a dict representing the number of translation available in a
+        languages in which there exist at least one translation in the state
+        object.
+
+        Returns:
+            dict(str, int). A dict with language code as a key and number of
+            translation available in that language as the value.
+        """
+        return self.written_translations.get_translation_counts()
+
+    def get_content_count(self):
+        """Returns the number of distinct content field available in the object.
```

Comment:
field --> fields

(I'm seeing this sort of error often, please correct it throughout).